src/conv/invokers/impl_gemm_dynamic.cpp

```diff
@@ -149,6 +253,34 @@ InvokerFactory MakeImplGemmDynamicForwardInvokerFactory(const ConvolutionContext
     };
 }
 
+InvokerFactory
+MakeImplGemmDynamicForwardInvokerFactory(const ConvolutionContext& ctx,
```

Comment:
ditto, maybe use a single `MakeImplGemmDynamicForwardInvokerFactory` not overloading

Reply:
ok

Reply:
ditto

Reply:
@carlushuang Could you pls re-review changes.